### DISCUSSION TITLE
ESTKALMAN: State out of bounds, resetting

### DISCUSSION THREAD

**Discussion by surendhar091992** [USC-ACTLab/crazyswarm#780] [Q&A]
I am using 4 cf2 in my experiment (simple cmdvel based waypoints), and i am using optitrack(flex-13) motion capture system. The moment i get the 'ESTKALMAN: State out of bounds, resetting' error, the cf2 crashes. i would like to know the reason for this particular error

I have attached the terminal error snapshot and (roll,pitch,yaw) from tf2 command.

Terminal error plot:
![err_plot](https://github.com/USC-ACTLab/crazyswarm/assets/104561072/4b126f64-ded9-43c6-80b3-8a647ff38d5b)

tf roll,pitch,raw plot:
![Screenshot from 2023-12-08 14-19-19](https://github.com/USC-ACTLab/crazyswarm/assets/104561072/da4e8389-541d-496a-91ac-dfa8f859ae47)

**Comment by whoenig** [comment]
This typically means that the optitrack data is not received by the drones. This is most of the time caused by a Crazyradio without broadcast support, i.e., make sure to upgrade your crazyradio firmware as instructed. It may also be caused by connectivity issues between Motive and ROS, but this should be more easy to rule out by checking the Rviz visualiztion.

**Reply by surendhar091992** [reply]
Thank you for getting back to me. I've have followed the instructions in crazyswarm-1 for the Crazyradio firmware. Additionally, are there other potential reasons for this error aside from Motive?

**Reply by whoenig** [reply]
No, those are the only two potential pitfalls. Does rviz show a movement?
